bitnami/aspnet-core/values.yaml

```diff
@@ -395,6 +395,61 @@ ingress:
   ##   key:
   ##   certificate:
 
+healthIngress:
+  ## Set to true to enable heallth ingress record generation
+  ##
+  enabled: false
+
+  ## Set this to true in order to add the corresponding annotations for cert-manager
+  ##
+  certManager: false
+
+  ## When the heallth ingress is enabled, a host pointing to this will be created
+  ##
+  hostname: aspnet-core.local
+
+  ## Ingress annotations done as key:value pairs
+  ## For a full list of possible ingress annotations, please see
+  ## ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md
+  ##
+  ## If certManager is set to true, annotation kubernetes.io/tls-acme: "true" will automatically be set
+  ##
+  annotations: {}
+
+  ## Enable TLS configuration for the hostname defined at ingress.hostname parameter
+  ## TLS certificates will be retrieved from a TLS secret with name: {{- printf "%s-tls" .Values.ingress.hostname }}
+  ## You can use the ingress.secrets parameter to create this TLS secret, relay on cert-manager to create it, or
+  ## let the chart create self-signed certificates for you
+  ##
+  tls: false
+
+  ## The list of additional hostnames to be covered with this heallth ingress record.
+  ## Most likely the hostname above will be enough, but in the event more hosts are needed, this is an array
+  ## extraHosts:
+  ## - name: aspnet-core.local
+  ##   path: /
+
+  ## The tls configuration for additional hostnames to be covered with this heallth ingress record.
+  ## see: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+  ## extraTls:
+  ## - hosts:
+  ##     - aspnet-core.local
+  ##   secretName: aspnet-core.local-tls
+
+  ## If you're providing your own certificates, please use this to add the certificates as secrets
+  ## key and certificate should start with -----BEGIN CERTIFICATE----- or -----BEGIN RSA PRIVATE KEY-----
+  ## name should line up with a secretName set further up
+  ##
+  ## If it is not set and you're using cert-manager, this is unneeded, as it will create the secret for you
+  ## If it is not set and you're NOT using cert-manager either, self-signed certificates will be created
+  ## It is also possible to create and manage the certificates outside of this helm chart
+  ## Please see README.md for more information
+  ##
+  secrets: []
+  ## - name: aspnet-core.local-tls
+  ##   key:
+  ##   certificate:
+
```

Comment:
You need to update the **values-production.yaml** too

Reply:
we need to  use the different annotations for health ingress. example **nginx.ingress.kubernetes.io/rewrite-target: /health/status** which not able to achieve in same ingress.
Updated the values-production.yaml